wondrous-app/components/Common/CornerWidget/Component.tsx

```diff
@@ -0,0 +1,122 @@
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import CloseModalIcon from 'components/Icons/closeModal';
+import get from 'lodash/get';
+import { useRouter } from 'next/router';
+
+import palette from 'theme/palette';
+import typography from 'theme/typography';
+import { ENTITIES_TYPES } from 'utils/constants';
+
+import { CornerWidgetComponentType } from './types';
+
+const addedToText = (type) => {
+  if (type === ENTITIES_TYPES.MILESTONE) return `launched in`;
+  return `added to`;
+};
+
+const useHandleOnClick = ({ id, type, orgName }) => {
+  const router = useRouter();
+  const entityLink = orgName ? `/organization/${router.query.username}/` : `/pod/${router.query.podId}/`;
```

Comment:
i think this is actually broken. if I create a task on mission control for example. or if I create a pod task on org page